velox/docs/functions/spark/aggregate.rst

```diff
@@ -7,6 +7,14 @@ Aggregate functions operate on a set of values to compute a single result.
 General Aggregate Functions
 ---------------------------
 
+.. spark:function:: avg(x) -> double|decimal
+
+    Returns the average (arithmetic mean) of all input values.
```

Comment:
> all input values

Should this be "all non-null input values"?

Reply:
Fixed presto and spark doc on this.